conf/st2.conf.sample

```diff
@@ -6,17 +6,21 @@
 enable = True
 
```

Comment:
That would be a bigger change and not part of this PR since it needs to be automated like this config.